gitlens/gl-gitkraken-browser-extension.md

```diff
@@ -0,0 +1,58 @@
+---
+
+title: GitKraken Browser Extension
+description: Learn all about the GitKraken Browser Extension.
+taxonomy:
+    category: gitlens
+
+---
+
+The GitKraken Browser Extension bridges your GitHub or GitLab repositories to GitKraken Client and GitLens in Visual Studio Code. With this browser extension enabled, you get options inside github.com and gitlab.com to open repositories and commits in GitKraken Client or GitLens in Visual Studio Code.
+
+<img src='/wp-content/uploads/gl-be-open-repo.gif' class='img-bordered img-responsive center'>
+
+***
+## Requirements
+
+* [Google Chrome](https://www.google.com/chrome/) or [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/new/)
+* [GitKraken Client](https://www.gitkraken.com/git-client/try-free) or [GitLens](https://www.gitkraken.com/gitlens/try-free) for Visual Studio Code
+
+***
+
+## How to install
+
+TODO
+
+***
+
+## Features
+
+Open with GitKraken: From GitHub's Code and GitLab’s Clone dropdowns, you can open a repository directly in GitKraken Client or with GitLens in Visual Studio Code.
+
+<img src="/wp-content/uploads/be-clone.png" class="img-bordered img-responsive center">
+
+Open Compairison in Visual Studio Code: From the pull request view on GitHub or GitLab, you can select "Open Comparison in VS Code" from the Code dropdowns for an easy transition into GitLens in Visual Studio Code.
+
+<img src="/wp-content/uploads/be-pull-request.png" class="img-bordered img-responsive center">
+
+Open with GitKraken button on commit pages: From a commit page on GitHub or GitLab, you can select "Open with GitKraken".
```

Comment:
Maybe a small extra bit to this sentence to specify what it does i.e. "to view the commit directly in GitKraken Client or with GitLens in Visual Studio Code." or something along those lines